testcases/testcase_blood_flow_model.py

```diff
@@ -20,7 +20,7 @@
 PARAMETERS = MappingProxyType(
     {
         # Setup parameters for blood flow model
-        "read_network_option": 1,  # 1: generate hexagonal graph
+        "read_network_option": 2,  # 1: generate hexagonal graph
```

Comment:
Please leave it as before.

Reply:
this is before the "before" you cite, when we merge the other it will be like that

Reply:
In the main branch, we would like to have the default parameters. Could you please just change this value to 1?